app/index.html

```diff
@@ -28,6 +28,7 @@
 
     {{content-for 'body'}}
 
+    <script>window.onerror=function(){document.body.innerHTML='<p style="width: 70%;background: var(--main-bg);padding: 10px;">Sorry, it looks like we were not able to load the page. Please make sure your network connection works and you are using an up-to-date browser. If the issue persists, please visit our <a href="https://github.com/rust-lang/crates.io/issues/new/choose">issue tracker</a> to report the problem.</p>'}</script>
```

Comment:
I think we should add a comment above this line, stating that any modifications to the inline script require an update to the CSP hash.

Reply:
added :)